documentation/src/main/asciidoc/migration/index.asciidoc

```diff
@@ -118,3 +118,187 @@ used to match no documents with the Lucene backend, but all documents with the E
 A boolean predicate with no clause will now consistently match no documents regardless of the backend.
 * API methods matching `*Async(..)` pattern (e.g. `SearchWorkspace#purgeAsync()`) will no longer block
 if internal queues of operations are full, but will throw `RejectedOperationException` instead.
+
+[[outboxpolling]]
+== Outbox polling system tables
+
+Due to changes to the entity that represents the outbox event, DB schema changes are required.
+A new column -  `created::timestamp` is introduced, `id` column changes the type from integer to varchar.
+You can find suggested migration scripts for the supported databases below:
```

Comment:
I know I used the term "supported", but I was taking liberties: we don't really provide support, and we don't even document which databases are supposed to work correctly... Let's not imply too much here.

```suggestion
You can find suggested migration scripts for the tested databases below:
```